snap/snap_run

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
```

Comment:
Please add the Apache license header to this file, as in https://github.com/apache/couchdb/blob/master/src/couch/priv/spawnkillable/couchspawnkillable.sh#L3-L17